src/vanilla/Model/ParameterTS.cs

```diff
@@ -1,51 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-// 
-
-using System.Collections.Generic;
-using System.Linq;
-using AutoRest.Core;
-using AutoRest.Core.Model;
-using Newtonsoft.Json;
-
-namespace AutoRest.TypeScript.Model
-{
-    public class ParameterTS : Parameter
-    {
-        public ParameterTS()
-        {
-            // if this is a client property, the name should be prefixed.
-            Name.OnGet += value =>
-                IsClientProperty
-                    ? false == Method?.MethodGroup?.IsCodeModelMethodGroup
-                        ? $"this.client.{ClientProperty.Name}"
-                        : $"this.{ClientProperty.Name}"
-                    : CodeNamer.Instance.GetParameterName(value);
-        }
-
-        [JsonIgnore]
-        public IEnumerable<Core.Model.Property> ComposedProperties
```

Comment:
This property didn't make sense, right?

Reply:
It didn't really make sense, but more importantly it wasn't used anywhere. That's the real reason I removed it.